schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+          authorizationCode:
+            allOf:
+              - $ref: '#CommonFlow'
+              - $ref: '#TokenFlow'
+              - $ref: '#AuthorizationFlow'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+        unevaluatedProperties: false
+        $defs:
+          Common:
+            $anchor: CommonFlow
+            properties:
+              refreshUrl:
+                type: string
+                format: uri-reference
+              scopes:
+                type: object
+                additionalProperties:
+                  type: string
+          Token:
+            $anchor: TokenFlow
+            required:
+              - tokenUrl
+            properties:
+              tokenUrl:
+                type: string
+                format: uri-reference
+          Authorization:
+            $anchor: AuthorizationFlow
+            required:
+              - authorizationUrl
+            properties:
+              authorizationUrl:
+                type: string
+                format: uri-reference
+      Link:
+        $anchor: Link
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            oneOf:
+              - properties:
+                  operationRef:
+                    type: string
+                    format: uri-reference
+                  operationId: false
+              - properties:
+                  operationRef: false
+                  operationId:
+                    type: string
+            properties:
+              parameters:
+                type: object
+                additionalProperties: true
+              requestBody: true
+              server:
+                $ref: '#Server'
+            unevaluatedProperties: false
+      Callback:
+        $anchor: Callback
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+      Schema:
+        $anchor: Schema
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: 'https://json-schema.org/draft/2020-12/schema'
+              - $ref: '#SchemaExtensions'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            unevaluatedProperties: false
+        $defs:
+          Extensions:
+            $anchor: SchemaExtensions
+            properties:
+              discriminator:
+                $ref: '#Discriminator'
+              externalDocs:
+                $ref: '#ExternalDocumentation'
+              xml:
+                $ref: '#XML'
```

Schema Object extensions are missing [`example`](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#user-content-schemaexample):
```suggestion
                $ref: '#XML'
              example:
                deprecated: true
```